src/adapters/postgres/user-adapter.ts

```
@@ -2966,6 +2969,263 @@ export class PostgresUserService implements IServicelocator {
+      LoggerUtil.error(
+        `${API_RESPONSES.SERVER_ERROR}`,
+        `Keycloak session invalidation failed for user ${user.userId}: ${logoutResult.message}`,
+        APIID.USER_ANONYMIZE
+      );
+    }
+
+    // 2. Postgres: overwrite exactly the agreed field set on the Users row.
+    await this.updateBasicUserDetails(user.userId, {
+      email: anonymizedIdentity,
+      username: anonymizedIdentity,
+      firstName: 'deleted',
```

ESLint

[error] Replace `'deleted'` with `"deleted"`

(prettier/prettier)